Assignment1-2 joshuayoung101/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+
+// This program calculates and displays the following
+// *Amount paid for stock
+// *Commission % paid to broker
+// *Amount sold for stock
+// *Amount paid to broker ($)
+// *Profit made
+
+
+int main() {
+    // Calculates and displays amount paid
+    double SHARES_BOUGHT = 1000;
+    double PURCHASE_PRICE = 45.50;
+    double AMOUNT_PAID = SHARES_BOUGHT*PURCHASE_PRICE;
+    std::cout << "Total amount paid for " << SHARES_BOUGHT << " stocks at $"
+            << PURCHASE_PRICE << " per stock = $" << AMOUNT_PAID <<std::endl;
+
+    // Calculates and displays commission on the purchase
+    double COMMISSION_RATE = .02;
```

Comment:
Some of these are constants and some are not.

The ones that hold the values listed in the assignment should have the `const` keyword the rest of the variables should be lowercase snake_case